query/graphql/pom.xml

```diff
@@ -97,9 +82,6 @@
                         <Bundle-SymbolicName>${project.artifactId}-beta</Bundle-SymbolicName>
                         <Service-Component />
                         <Embed-Dependency>
-                            ddf-security-common,
-                            platform-util,
-                            platform-util-unavailableurls,
```

Comment:
Can the `maxArtifactSize` of the `ArtifactSizeEnforcerRule` be lowered after removing these?